client/my-sites/site-settings/featured-image-toggle.jsx

```diff
@@ -24,7 +25,14 @@ const FeaturedImageTemplateToggle = ( props ) => {
 				/>
 				<FormSettingExplanation>
 					{ translate(
-						"Includes your post's featured image in the email sent out to your readers."
+						"Includes your post's featured image in the email sent out to your readers. {{link}}Learn more about the featured image{{/link}}.",
+						{
+							components: {
+								link: (
+									<a href={ localizeUrl( 'https://wordpress.com/support/featured-images/' ) } />
```

Comment:
```suggestion
									<a href={ localizeUrl( 'https://wordpress.com/support/featured-images/' ) } target="_blank" rel="noreferrer" />
```

Wondering if it would be better to open in a new tab with noreferrer set? I'm seeing practices like that in similar "Learn more" links in the Calypso settings.

Reply:
> Wondering if it would be better to open in a new tab with noreferrer set? I'm seeing practices like that in similar "Learn more" links in the Calypso settings.

Good point! 

I was considering opening the link in a new tab for a moment, but then saw Feed setting description opening in the same tab:

![Markup on 2022-12-08 at 15:09:51](https://user-images.githubusercontent.com/25105483/206467660-e2c1e429-ae7e-46a6-bdee-3a4d7269c044.png)

Now reading your comment I went to look at other setting pages and found these CDN settings for instance - that open in a new tab:

![Markup on 2022-12-08 at 15:08:40](https://user-images.githubusercontent.com/25105483/206467837-789ad010-b14a-4ccb-9b02-fca51b2e3d5e.png)

I would personally voted for opening in the same tab - as the user can then decide whether they want to open the link in a new tab or not.

At the same time, from other perspective it seems better to open the link in a new tab - since the link leads outside of the interface into our docs. I think that user who wants to learn more about a specific setting doesn't want to go away from their dashboard and could prefer having the docs opened in a new tab.

Another interesting approach could be opening the docs in an overlay window like this (used for Testimonials settings for instance):

https://user-images.githubusercontent.com/25105483/206470075-ca43034c-0946-4b1b-a363-e68c18be8a2c.mp4

Do you have a preference here @saygunnyc?

Reply:
While the modal/overlay window is interesting –something we could explore with the Help Center efforts cc @nuriapenya @poligilad-auto, I'd say let's have it open up in a new window, so the user doesn't have to leave the Settings page and remain in context.

Reply:
> I'd say let's have it open up in a new window, so the user doesn't have to leave the Settings page and remain in context.

Perfect! Thanks for sharing your perspective. I will apply the change in the next commit.

One more thing: What do think about Yan's copy-change suggestion? https://github.com/Automattic/wp-calypso/pull/70870#discussion_r1043245649

Reply:
The link is now opening in a new tab (https://github.com/Automattic/wp-calypso/pull/70870/commits/82c7be79afb166e5521894e5a0d763122c55c726).